js/screens/sensor-setup.js

```diff
@@ -85,31 +85,68 @@ class SensorSetup extends Component {
     thing.active = 1;
     thing.status = 'ACTIVATED';
 
+    let user = null;
+    let children = [];
+
     AuthService.getUser().then((user) => {
         thing.user_id = user.id;
+        user = user;
+        return user;
     }).then(() => {
       return Promise.all([
         PlatformService.addThing(thing),
         PlatformService.getTypeChannels(this.state.sensorType.id)
       ]);
     }).then((result) => {
-      let deviceId = result[0].id;
+      thing.id = result[0].id;
       promises = _.map(result[1], function(channel){
         return PlatformService.addThing({
           name: channel.name,
-          parent_id: deviceId,
+          parent_id: thing.id,
           active: 1,
           status: 'ACTIVATED',
           device_type_id: '8f93f0f0-db59-44c1-aaa4-bd48707de97b',
           properties: {
             channel: channel.channel
           }
+        }).then((child) => {
+          children.push(child);
         });
       });
+      return Promise.all(promises)
+    }).then(() => {
+      return RulesService.getDeviceTypeRules(thing.device_type_id);
+    }).then((deviceRules) => {
+      var promises = _.map(deviceRules, function(ruleTemplate){
+
+        let notification = {
+          "data" : {
+            "owner" : user.first_name + ' ' + owner.last_name,
+            "device_name" : thing.name,
+          },
+          "value" : user.email,
+          "method" : "email",
+          "type" : "msm-cayenne-notification",
```

Comment:
Need to have some logic to decide if this is a sensor value alert or online/offline alert. 
for sensor value alerts use "msm-cayenne-notification" for online/offline alerts use "msm-cayenne-offline"